### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by mecoreyg** [maxileith/homebridge-appletv-enhanced#247] [General]
First of all thank you! I have been looking for this for months and really didn't want to move all my stuff to Home Assistant. So I am super happy and thankful for this development.

I am confused on the wording of some of this. Like for instance, what is Set-top box. What is this useful for? What I want to do is when anything is playing, turn off lights and when anything is idle (paused, tv off, sitting at Home Screen, etc) have lights on low.

Also, for the Switches and Motion Types, there are no headings so I don't know which is which

**Comment by maxileith** [comment]
> I am confused on the wording of some of this. Like for instance, what is Set-top box. What is this useful for?

see https://github.com/maxileith/homebridge-appletv-enhanced/issues/191.

> What I want to do is when anything is playing, turn off lights and when anything is idle (paused, tv off, sitting at Home Screen, etc) have lights on low.

You could do two simple automations.

1. When the "Playing" motion sensor detects motion, turn off the lights
2. When the "Playing" motion sensor stops to detect motion, turn on the lights at a low brightness

> Also, for the Switches and Motion Types, there are no headings so I don't know which is which

By default, the motions sensors are named like the state itself ... Playing, Stopped, Loading, Seeking, ...

**Reply by mecoreyg** [reply]
@maxileith - for the set box.. I just don't get what it means by "set-top box". Like what is the explanation of this feature? What do you mean by set-top box accessory?

I will look into the automations with motion, thank you.

**Reply by maxileith** [reply]
@mecoreyg it is just the type of the accessory that is exposed by the plugin. When using the `setTopBox` option, the Apple TV is exposed as a Set-Top Box instead of an Apple TV. You do not need to use the `setTopBox` option.

**Reply by mecoreyg** [reply]
@maxileith okay thanks! Automations works and the plugin does exactly what I need. THANKS to you and your team for the development of this. I will promote it

**Reply by maxileith** [reply]
I am the Team 😅
